src/Winton.Extensions.Configuration.Consul/Winton.Extensions.Configuration.Consul.csproj

```diff
@@ -6,7 +6,7 @@
     <Copyright>Copyright 2018 Winton</Copyright>
     <Description>Provides support for configuring .NET Core applications with Consul</Description>
     <GenerateDocumentationFile>True</GenerateDocumentationFile>
-    <NoWarn>$(NoWarn);SA1101;SA1309;SA1413;</NoWarn>
+    <NoWarn>$(NoWarn);S112;SA1101;SA1309;SA1413;</NoWarn>
```

Comment:
Was this intentional? I can't find a reference to warning with this code and it looks like a possible typo of of an `SAXXXX` type code.

Reply:
See my other comment (this is from `SonarLint`)